src/zenml/models/v2/core/step_run.py

```diff
@@ -331,7 +330,7 @@ def inputs(self) -> Dict[str, "ArtifactVersionResponse"]:
         return self.get_body().inputs
 
     @property
-    def outputs(self) -> Dict[str, "ArtifactVersionResponse"]:
+    def outputs(self) -> Dict[str, List["ArtifactVersionResponse"]]:
```

Comment:
I'm also wondering whether we might want to separate this into `regular outputs` from the step, and manual ones (using `save_artifact` and `register_artifact`)